www/web-proxy-useracl/src/opnsense/www/css/OPNsense/ProxyUserACL/bootstrap-datetimepicker.css

```diff
@@ -0,0 +1,374 @@
+/*!
+ * Datetimepicker for Bootstrap 3
+ * version : 4.17.45
+ * https://github.com/Eonasdan/bootstrap-datetimepicker/
```

Comment:
this looks like it is EoL (from the README). Is there a big advantage over using the browsers functionality?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

Just in case: if this is added, it must go to core because it must be part of the theme as some colors etc. may have to be overridden by themes.